source/language/types.rst

```diff
@@ -279,25 +279,53 @@ their argument expression in parentheses:
 
    .. table:: Built-in mathematical functions in OpenQASM3.
 
-      +--------+
-      | arccos |
-      +--------+
-      | arcsin |
-      +--------+
-      | arctan |
-      +--------+
-      | cos    |
-      +--------+
-      | exp    |
-      +--------+
-      | ln     |
-      +--------+
-      | sin    |
-      +--------+
-      | sqrt   |
-      +--------+
-      | tan    |
-      +--------+
+      +----------+--------------------------------+---------------------+
+      | Function | [Input Range/Type] , [...]     | [Output Range/Type] |
+      +==========+================================+=====================+
+      | arccos   | [-1, 1]                        | [0, pi rad]         |
+      +----------+--------------------------------+---------------------+
+      | arcsin   | [-1, 1]                        | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | arctan   | [-∞, Real, ∞]                  | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | ceiling  | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | cos      | [float]                        | [float]             |
```

Comment:
Is there a reason for the type of `cos` being `float`, but of `tan` being `angle`?  I'd generally expect them both to accept both and return `float`.

Reply:
Makes sense to me, which isn't of itself definitive, but helps me edit :)

Reply:
Done.